source/_integrations/matter.markdown

```diff
@@ -23,182 +23,241 @@ ha_platforms:
+A lot of devices that (will) hit the market will use Thread for the radio communication and Matter as control protocol but this is not guaranteed. There are for example also Thread based devices available that only support Apple HomeKit or some vendor specific communication protocol. There are also a few  cases where you need to apply for a (beta) firmware update on the device to enable Matter as communication protocol. Therefore, whenever you are looking for devices, do not assume Matter support when you see a Thread logo. Look for the *Matter* logo itself (on either Wi-Fi/Ethernet based devices or Thread) or any other confirmation by the manufacturer that the device supports Matter.
 </p>
 
-One of the great features of Matter is the so called _Multi Fabric_ feature: you can join the same device to multiple controllers. For example: add it to Google Home, Apple Home, and Home Assistant at the same time.
+## Bluetooth
 
-For devices where Home Assistant provides a native integration (with local API), Matter may not be the best option. Matter, being a universal standard, might not have the nitty-gritty features that come with a product specific protocol. A good example is Philips Hue:  the communication over Matter only provides the basic controls over lights, the official integration brings all Hue unique features like (dynamic) scenes, entertainment mode, etc.
+Most (if not all) Matter compliant devices will also have a Bluetooth chip onboard, this is to ease commissioning (a somewhat technical term for adding a device to your controller). Bluetooth will not be used for controlling a device but only for pairing it after unboxing or factory reset. The Home Assistant controller uses the Home Assistant Companion app to do commissioning so you can bring your phone close to the device you want to commission. Your network credentials will then be sent by the controller to your device over Bluetooth in the commissioning process. If that succeeded, the device will furthermore communicate over its native interface, meaning WiFi, Ethernet or Thread.
 
 <p class='note'>
-The Matter protocol relies on (local) IPv6 and mDNS (multicast traffic) which should be able to travel freely in your network. Matter devices (and any Thread Border routers) must be on the same LAN/VLAN as Home Assistant. Implementations like mDNS reflectors usually do more harm than good.
+Although your Home Assistant server might have a Bluetooth adapter on board that the controller can use for commissioning of devices, we choose not to utilize that adapter. Mainly to prevent issues with the built-in Bluetooth integration but also because it makes more sense to bring your mobile devices close to the Matter device you'd like to commission.
```

Comment:
```suggestion
Although your Home Assistant server might have a Bluetooth adapter on board that the controller can use to commission devices, we choose not to utilize that adapter. Mainly to prevent issues with the built-in Bluetooth integration but also because it makes more sense to bring your mobile devices close to the Matter device you'd like to commission.
```